content/maintainers-guide/how-to-become-a-committer.md

```diff
@@ -40,9 +34,8 @@ All the above, plus:
 ## How to apply to be a committer
 
 1. [Open an issue on the open source repository](https://github.com/PrestaShop/open-source/issues/new) explaining why you apply to become a committer, and on what part of the project (Core or module/subproject).
-2. Discuss with current maintainers on this issue; particularly, this will be the time for maintainers to analyze the candidate's motivation and skills on the different parts of the project that they applied for.
-3. Once all questions have been answered, current maintainers vote.
-4. If the simple majority of positive votes is achieved, the new committer is approved.
+2. If three maintainers approve the application, the new committer is approved.
+3. After one week, if the issue did not receive three positive votes, it is considered rejected and closed.
```

Comment:
One week might be too short (eg. if people are too busy or on vacation)

Reply:
30 days then